src/lib.rs

```diff
@@ -210,25 +211,25 @@ impl EguiMq {
         }
 
         if cursor_icon == egui::CursorIcon::None {
-            mq_ctx.show_mouse(false);
+            miniquad::window::show_mouse(false);
         } else {
-            mq_ctx.show_mouse(true);
+            miniquad::window::show_mouse(true);
 
             let mq_cursor_icon = to_mq_cursor_icon(cursor_icon);
             let mq_cursor_icon = mq_cursor_icon.unwrap_or(mq::CursorIcon::Default);
-            mq_ctx.set_mouse_cursor(mq_cursor_icon);
+            miniquad::window::set_mouse_cursor(mq_cursor_icon);
         }
 
         if !copied_text.is_empty() {
-            self.set_clipboard(mq_ctx, copied_text);
+            self.set_clipboard(copied_text);
         }
     }
 
     /// Call this when you need to draw egui.
     /// Must be called after `end_frame`.
-    pub fn draw(&mut self, mq_ctx: &mut mq::Context) {
+    pub fn draw(&mut self, mq_ctx: &mut dyn mq::RenderingBackend) {
         if let Some(shapes) = self.shapes.take() {
-            let meshes = self.egui_ctx.tessellate(shapes);
+            let meshes = self.egui_ctx.tessellate(shapes, self.native_dpi_scale);
```

Comment:
Here we should use the `pixels_per_point` reported by egui, because that can differ from `native_dpi_scale` if the user has bit Ctrl+/-